coalib/coala_main.py

```diff
@@ -63,7 +62,6 @@ def run_coala(console_printer=None,
     :return:                        A dictionary containing a list of results
                                     for all analyzed sections as key.
     """
-    configure_logging()
```

Comment:
I think this one could also go into a separate commit, there you can also put your explanation into the commit body :+1: